taxi_demo/summary.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "|<img src=\"https://patch.com/img/cdn20/users/22965231/20180313/040305/styles/raw/public/processed_images/shutterstock_110928791-1520971084-4672.jpg\" width=\"200\" /> | <img src=\"https://pbs.twimg.com/media/EeBsCTPX0AAICkt.png\" width=\"400\" />|\n",
+    "| -- | -- |\n",
+    "\n",
+    "# NYC Taxi analysis with Saturn Cloud\n",
+    "\n",
+    "The notebooks in this repo showcase an end-to-end data science workflow executed on [Saturn Cloud](https://www.saturncloud.io/).\n",
+    "\n",
+    "> Saturn Cloud is a data science and machine learning platform for scalable Python analytics. It automates DevOps and ML infrastructure with cloud-based Jupyter and Dask for big data, while leveraging Docker and Kubernetes so that your work is reproducible, shareable, and ready for production. \n",
+    "\n",
+    "The following tasks are performed:\n",
+    "\n",
+    "1. Ingest 1.6 billion CSV records and write to optimized Parquet files\n",
+    "1. Create train/test sets for machine learning tasks\n",
+    "1. Train a number of machine learning models\n",
+    "1. Deploy models via REST API\n",
+    "1. Create dashboard with exploratory analysis and ML model diagnostics\n",
+    "\n",
+    "The dashboard features a live-scoring widget using a model hosted with Saturn Cloud.\n",
+    "\n",
+    "<img src=\"img/pipeline.png\" width=\"800\">\n",
+    "\n",
+    "If you don't already use Saturn Cloud, [see how to get started here](https://www.saturncloud.io/docs/getting-started/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dashboard\n",
+    "\n",
+    "The dashboard is hosted using a Saturn Cloud [Deployment](https://www.saturncloud.io/docs/deployments/) and presents analysis of ride volume, fare collections, and a detailed analysis of driver tips. The \"ML\" tab shows various diagnostics for the machine learning models, including metric summaries and longidtudinal analysis for model drift detection. It also features a live-scoring widget using a model also hosted with a Saturn Deployment.\n",
+    "\n",
+    "![](img/dashboard1.png)\n",
+    "\n",
+    "![](img/dashboard2.png)\n",
+    "\n",
+    "![](img/dashboard3.png)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import dask.dataframe as dd\n",
+    "import holoviews as hv\n",
+    "import hvplot.pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
This cell is not needed.

Reply:
it doesnt render on github, but this left-justifies the table, which i like in jupyterlab

Reply:
im referring to markdown here, not cell output